File: Python3/프로그래머스/0/120831. 짝수의 합/짝수의 합.py
def solution(n):
    answer = 0  # 더한 값을 저장할 변수 0으로 초기화
    for i in range(2, n + 1, 2):  # 짝수는 2에서 시작해서 2씩 증가합니다. 그리고 n이 짝수이면 n도 포함해야 합니다
        answer += i               # 기존 합에 새로운 짝수 더하기
    return answer

File: C++/프로그래머스/0/120831. 짝수의 합/짝수의 합.cpp
using namespace std;

int solution(int n) {
    if (n % 2 != 0)
    {
        n = n - 1;
    }

    int answer = 0;
    while (n > 0)
    {
        answer += n;
        n -= 2;
    }
    return answer;
}